HIPs/hip-1.md

```diff
@@ -0,0 +1,271 @@
+- Specification - The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Handshake platforms (HSD, RSD)
+- Rationale - The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.
+- Backwards Compatibility - All HIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The HIP must explain how the author proposes to deal with these incompatibilities. HIP submissions without a sufficient backwards compatibility treatise may be rejected outright.
+- Test Cases - Test cases for an implementation are mandatory for HIPs that are affecting consensus changes. Other HIPs can choose to include links to test cases if applicable.
+- Implementations - The implementations must be completed before any HIP is given status “Final”, but it need not be completed before the HIP is merged as draft. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of “rough consensus and running code” is still useful when it comes to resolving many discussions of API details.
+- Security Considerations - All HIPs must contain a section that discusses the security implications/considerations relevant to the proposed change. Include information that might be important for security discussions, surfaces risks and can be used throughout the life cycle of the proposal. E.g. include security-relevant design decisions, concerns, important discussions, implementation-specific guidance and pitfalls, an outline of threats and risks and how they are being addressed. HIP submissions missing the "Security Considerations" section will be rejected. An HIP cannot proceed to status "Final" without a Security Considerations discussion deemed sufficient by the reviewers.
+- Copyright Waiver - All HIPs must be in the public domain. See the bottom of this HIP for an example copyright waiver.
+
+## HIP Formats and Templates
+
+HIPs should be written in [markdown] format.
+Image files should be included in a subdirectory of the `assets` folder for that HIP as follows: `assets/HIP-N` (where **N** is to be replaced with the HIP number). When linking to an image in the HIP, use relative links such as `../assets/HIP-1/image.png`.
```

Comment:
you can create an empty assets folder for your PR so at least something exists. Maybe just include a small png of the handshake logo (github wont like a totally empty folder)